core/domain/exp_services.py

```diff
@@ -1498,6 +1520,15 @@ def save_new_exploration_from_yaml_and_assets(
         raise Exception('Invalid YAML file: missing schema version')
     exp_schema_version = yaml_dict['schema_version']
 
+    # The assets are commited before the exploration is created because the
+    # migrating to state schema version 24 involves adding dimensions to images
+    # .So we need to have images in the datastore before we could perform the
```

Comment:
Nit: move the period to the end of the previous line, or move the last word of the previous line to here.

Add a space after the period and before "So".

Reply:
Done